packages/test/harness/src/host/dev/hostOverrides/done.js

```diff
@@ -1,14 +1,11 @@
 // In dev mode, draw a green tick when test succeeded.
 
-const checkAccessibility = require('../../common/host/checkAccessibility');
 const dumpLogs = require('../../common/dumpLogs');
 const override = require('../utils/override');
 
 // Send the completion back to the browser console.
 module.exports = (webDriver, done) =>
   override(done, undefined, async () => {
-    await checkAccessibility(webDriver)();
```

Comment:
Do we still check accessibility?

Reply:
Yes, we still check it in the default place.

This is on the `post()` of "hostOverrides", which means, run after the original `done()`. The original `done()` do the check, plus it look at if `runOptions.skipCheckAccessibility`.